backstop/tests/side-navigation.html

```diff
@@ -95,131 +103,261 @@ <h1>Side navigation</h1>
+                role="tab"
+                aria-selected="false"
+                aria-label="Calendar"
+                aria-controls="calendar-tab"
+                id="test-tab-1"
+              >
+                <svg-calendar
+                  class="iui-icon"
+                  aria-hidden="true"
+                ></svg-calendar>
+                <span class="iui-label">Calendar</span>
+              </button>
+              <span class="iui-tooltip">Calendar</span>
+            </div>
+
+            <div class="iui-bottom">
+              <button
+                class="iui-button iui-large iui-sidenav-button"
+                role="tab"
+                aria-selected="false"
+                aria-label="Administration"
+                aria-controls="administration-tab"
+                id="test-tab-2"
+              >
+                <svg-setup
+                  class="iui-icon"
+                  aria-hidden="true"
+                ></svg-setup>
+                <span class="iui-label">Administration</span>
+              </button>
+              <span
+                class="iui-tooltip"
+                style="margin-bottom: 10px;"
+              >Administration</span>
+            </div>
           </div>
 
-          <div class="iui-bottom">
-            <button
-              class="iui-button iui-large iui-sidenav-button"
-              role="tab"
-              aria-selected="false"
-              aria-label="Administration"
-              aria-controls="administration-tab"
-              id="test-tab-2"
+          <!-- Expand arrow -->
+          <button
+            class="iui-button iui-expand iui-sidenav-button"
+            role="button"
+            aria-label="Expand menu"
+            onclick="handleHidePanel()"
+            id="test-expand"
+          >
+            <!-- Use chevron-right.svg from @itwin/itwinui-icons package. -->
+            <svg
+              class="iui-icon"
+              aria-hidden="true"
+              viewBox="0 0 16 16"
             >
-              <svg-setup
-                class="iui-icon"
-                aria-hidden="true"
-              ></svg-setup>
-              <span class="iui-label">Administration</span>
-            </button>
-            <span
-              class="iui-tooltip"
-              style="margin-bottom: 10px;"
-            >Administration</span>
+              <path d="m4.7 0-1.4 1.4 6.6 6.6-6.6 6.6 1.4 1.4 8-8z" />
+            </svg>
+          </button>
+        </div>
+
+        <div
+          class="iui-side-navigation-submenu"
+          id="myOpeningSubMenu"
+        >
+          <div class="iui-submenu-header">
+            <div class="iui-submenu-header-label">
+              <button class="iui-button iui-borderless">
+                <svg-chevron-left
+                  class="iui-icon"
+                  aria-hidden="true"
+                ></svg-chevron-left>
+              </button>
+              <span id="demo-take-hover">Documents</span>
```

Comment:
Can we have here a longer string? Just to make sure that ellipsis is working.

Reply:
Yes sir.
![Screen Shot 2021-09-21 at 9 25 04 AM](https://user-images.githubusercontent.com/849817/134178842-1064cc3b-cdc3-4e3d-a734-abf467e16545.png)